Neos.FluidAdaptor/Classes/ViewHelpers/Format/JsonViewHelper.php

```diff
@@ -89,4 +91,26 @@ public static function renderStatic(array $arguments, \Closure $renderChildrenCl
 
         return json_encode($value, $options);
     }
+
+    /**
+     * @param string $argumentsName
+     * @param string $closureName
+     * @param string $initializationPhpCode
+     * @param ViewHelperNode $node
+     * @param TemplateCompiler $compiler
+     * @return string
+     */
+    public function compile($argumentsName, $closureName, &$initializationPhpCode, ViewHelperNode $node, TemplateCompiler $compiler)
```

Comment:
is the renderStatic method still needed? e.g. it was removed & replaced in the base64 view helper